core/embed/bootloader/emulator.c

```diff
@@ -44,26 +74,61 @@ __attribute__((noreturn)) int main(int argc, char **argv) {
     (void)ret;
   }
 
-  if (argc == 2) {
-    if (argv[1][0] == 's') {
-      // Run the firmware
-      stay_in_bootloader_flag = STAY_IN_BOOTLOADER_FLAG;
-    }
+  int opt;
+  bool display_error = false;
+  uint8_t set_variant = 0xff;
+  uint8_t color_variant = 0;
+  uint8_t bitcoin_only = 0;
```

Comment:
The "OTP" in emulator is not persistent, if that's what you mean. it only lives in RAM. It would be simple enough to add `$PROFILE_DIR/trezor.otp` or something, but that's not what I'm doing here.

Reply:
ah ok, i thought i was already stored in a file. lets do this when/if we need this.